Signal/src/ViewControllers/OWSViewController.h

```diff
@@ -15,6 +15,8 @@ NS_ASSUME_NONNULL_BEGIN
 // Use this method in lieu of autoPinToBottomLayoutGuideOfViewController:
 - (void)autoPinViewToBottomGuideOrKeyboard:(UIView *)view;
 
+- (void)setShouldIgnoreKeyboardChanges:(BOOL)value;
```

Comment:
TIOLI: consider making this a public, writable, property

Reply:
Done.